src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle() {
+
+    }
+
+    public Circle(double radius, String color) {
+        this.radius = radius;
+        setColor(color);
+        setFigureName("circle");
```

Comment:
Don't use setters inside the constructor. Let's create a constructor in the Figure class and call it.

Reply:
done